cogs/annual_handover_and_reset.py

```diff
@@ -0,0 +1,198 @@
+        """
+        # NOTE: Shortcut accessors are placed at the top of the function, so that the exceptions they raise are displayed before any further errors may be sent
+        main_guild: discord.Guild = self.bot.main_guild
+        committee_role: discord.Role = await self.bot.committee_role
+        committee_elect_role: discord.Role = await self.bot.committee_elect_role
+
+        handover_channel: AllChannelTypes | None = discord.utils.get(
+            main_guild.channels,
+            name="Handover",
+        )
+
+        initial_response: discord.Interaction | discord.WebhookMessage = await ctx.respond(
+            ":hourglass: Running handover procedures... :hourglass:",
+        )
+        logger.debug("Running the handover command!")
+
+        if main_guild.me.top_role.position < committee_role.position:
+            logger.debug(
+                "Handover command aborted because the bot did not "
+                "hold a role above the committee role.",
+            )
+            await initial_response.edit(
+                content=":warning: This command requires the bot to hold a role higher than "
+                "that of the committee role to perform this action. Operation aborted. "
+                ":warning:",
+            )
+            return
+
+        category: discord.CategoryChannel
+        for category in main_guild.categories:
+            if "committee" in category.name.lower() and "archive" not in category.name.lower():
+                await initial_response.edit(
+                    content=f":hourglass: Updating channels in category: {category.name} "
+                    ":hourglass:",
+                )
+                for channel in category.channels:
+                    logger.debug("Resetting channel permissions for channel: %s", channel)
+                    await channel.set_permissions(committee_elect_role, overwrite=None)
+
+        await initial_response.edit(
+            content=":hourglass: Giving committee members access to #handover and "
+            "removing committee role... :hourglass:",
+        )
+
+        committee_member: discord.Member
+        for committee_member in committee_role.members:
+
+            if isinstance(handover_channel, discord.TextChannel):
+                logger.debug("Giving user: %s, access to #handover", committee_member)
+                await handover_channel.set_permissions(
+                    committee_member,
+                    read_messages=True,
+                    send_messages=True,
+                )
+
+            logger.debug("Removing Committee role from user: %s", committee_member)
+            await committee_member.remove_roles(
+                committee_role,
+                reason=f"{ctx.user} used TeX-Bot slash-command: \"handover\"",
+            )
+
+        await initial_response.edit(
+            content=":hourglass: Giving committee-elect Committee role and "
+            "removing committee-elect... :hourglass:",
```

Comment:
Multi-line strings should be separated with brackets & response message can be improved.

Fix:
```python
await initial_response.edit(
    content=(
        ":hourglass: Giving committee-elect users the "Committee" role "
        "and removing their "Committee-Elect" role... :hourglass:"
    ),
)
```